includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -704,6 +704,9 @@ private function sanitize_disallowed_attributes_in_node( $node, $attr_spec_list
 
 		foreach ( $attrs_to_remove as $attr ) {
 			$node->removeAttributeNode( $attr );
+			if ( isset( $this->args['mutation_callback'], $attr->name ) ) {
+				call_user_func( $this->args['mutation_callback'], $node, 'removed_attr', $attr->name );
```

Comment:
I believe `AMP_Mutation_Util::ATTRIBUTE_REMOVED` could be used here.

Reply:
Thanks, this [commit](https://github.com/Automattic/amp-wp/commit/69317b877679ceb4bfb4a319244de0995f1b1222) from above substitutes the constant.